.github/dependabot.yml

```diff
@@ -3,7 +3,8 @@ updates:
   - package-ecosystem: gomod
     directory: /
     schedule:
-      interval: daily
+      interval: weekly
+      day: sunday
     open-pull-requests-limit: 5
```

Comment:
If we do the update on a weekly basis, I'd bump the limit to 10 or 15 or entirely remove it and remove the monthly update. We have so many dependencies that dependabot probably won't be able to catch up otherwise.

Reply:
oops sorry robin i think i missed this suggestion. just to double check, your suggestion is to remove the monthly update for:

```
- dependency-name: "github.com/aliyun/alibaba-cloud-sdk-go"
- dependency-name: "github.com/aws/*"
- dependency-name: "github.com/Azure/*"
```

and update them on weekly basis with all the other dependencies, right?